recipes/qtxlsxwriter/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os, glob, shutil
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain
+from conans import ConanFile, tools
+
+
+class QtXlsxWriterConan(ConanFile):
+    name = "qtxlsxwriter"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dbzhang800/QtXlsxWriter"
+    description = "A library that can read and write Excel files"
+    topics = ("qtxlsxwriter", "excel", "xlsx", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    generators = "CMakeDeps"
+    exports_sources = [
+        "CMakeLists.txt",
+        os.path.join("cmake", "*"),
+        os.path.join("patches", "*"),
+        os.path.join("src", "CMakeLists.txt")
+    ]
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("qt/5.15.2", private=False)
```

Comment:
not required in CCI

```suggestion
        self.requires("qt/5.15.2")
```

Reply:
Fixed by [932a3d7](https://github.com/conan-io/conan-center-index/pull/4838/commits/932a3d72b980f25d9b2fa72a2d42d5708c62fce4)